source/libs/utils.js

```diff
@@ -18,7 +18,7 @@ export const getRepoPath = () => {
 		return getCleanPathname().split('/').slice(2).join('/');
 	}
 
-	return false;
+	return '';
```

Comment:
In all the places `getRepoPath` was being used, a string was expected, so I couldn't see any reason why you were returning false, so I return an empty string now.

Reply:
The reason is when this function is called outside a repo. If it’s not a repo, there’s no “repo path”, hence its false. 

A `''` value means we are in the repo root

Reply:
The other PR fixed this by returning undefined, so this becomes `boolean?`